scripts/gen-new-employee-key.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Check if the correct number of arguments are provided
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <KEY_NAME> <KEY_EMAIL>"
+    exit 1
+fi
+
+# Variables from command line
+KEY_NAME="$1"
+KEY_EMAIL="$2"
+KEY_COMMENT="Employee Cert Key"
+KEY_FILE="$KEY_EMAIL.asc"
+REVOCATION_FILE="$KEY_EMAIL-revocation.asc"
```

Comment:
@pietrushnic this variable is not used, is it intended?

Reply:
Fixed in 4bc96d06bf29